### PR TITLE
Adding ExtensionImage + Aaz + Assess-Patches

### PR DESCRIPTION
**Pull request by ankjayant**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

First time using connectedmachine with aaz-dev-tools. We generated existing commands and resources as well two new commands. az connectedmahcine extensionimage and az connectedmahcine assess-patches. 

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by ankjayant** [comment]
> Hi @ankjayant, If you want to release the new extension version. Please write the description of changes into HISTORY.rst and update setup.py.

Updated these files

**Comment by ankjayant** [comment]
> Thank you for your contribution! We will review the pull request and get back to you soon.

Hey - still looking for a review

File: src/connectedmachine/setup.py
Comment (kairu-ms) [inline src/connectedmachine/setup.py:13]: Why skip version 0.6.0?

File: src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/_install_patches.py
Comment (kairu-ms) [inline src/connectedmachine/azext_connectedmachine/aaz/latest/connectedmachine/_install_patches.py:18]: Please add examples for these commands in aaz-dev-tools

**Review by kairu-ms — CHANGES_REQUESTED** [review]
Please add tests for those new commands

File: src/connectedmachine/setup.py
Comment (ankjayant) [reply src/connectedmachine/setup.py:13]: fixed this

**Comment by kairu-ms** [comment]
@ankjayant, it seems you removed the tests files.

**Comment by yaotongms** [comment]
> @ankjayant, it seems you removed the tests files.

Hello Kairu, my name is Yao. Ankit is no longer in Microsoft, and I will be taking over his work. I'm trying to merge this PR into azure cli extension and saw your comment on adding test files to the commands. I have some questions that I hope you can answer.

- Can I just add back all the tests he deleted?
- If not, do I need to start over from exporting the command tree to aaz repo, generate the cli commands and then start a new PR?

**Comment by kairu-ms** [comment]
> > @ankjayant, it seems you removed the tests files.
> 
> Hello Kairu, my name is Yao. Ankit is no longer in Microsoft, and I will be taking over his work. I'm trying to merge this PR into azure cli extension and saw your comment on adding test files to the commands. I have some questions that I hope you can answer.
> 
> * Can I just add back all the tests he deleted?
> * If not, do I need to start over from exporting the command tree to aaz repo, generate the cli commands and then start a new PR?
> * I registered my github account with my microsoft email but don't have permission to push changes to aaz or az-cli-extension. I wonder if I need to obtain permission from someone.

**Comment by kairu-ms** [comment]
> > @ankjayant, it seems you removed the tests files.
> 
> Hello Kairu, my name is Yao. Ankit is no longer in Microsoft, and I will be taking over his work. I'm trying to merge this PR into azure cli extension and saw your comment on adding test files to the commands. I have some questions that I hope you can answer.
> 
> * Can I just add back all the tests he deleted?
> * If not, do I need to start over from exporting the command tree to aaz repo, generate the cli commands and then start a new PR?
> * I registered my github account with my microsoft email but don't have permission to push changes to aaz or az-cli-extension. I wonder if I need to obtain permission from someone.

Hi @yaotongms
* This PR is not merged. So you can add back all the tests from the main branch or from the previous commits.
* @ankjayant has submit his change of aaz-model in this PR https://github.com/Azure/aaz/pull/189, you can pull his change and continue.
* The aaz and azure-cli-extensions repos are not allowed to push directly. You should fork these repo and push changes in your forked repo. Then create a PR from your forked repo to these repos

**Comment by yaotongms** [comment]
> 

Hello Kairu, thanks for the info! 

If I understand it correctly, this PR needs tests to be added back. Are those generated when generating the cli command on the aaz ui?

Moreover, Ankit's aaz repo seems to have some conflicts that needs to be resolved and need another reviewer to review it. I don't seem to have the write access to it so I'm not sure what the conflict is.

If it's more convenient, I can generate the extensionimage command again and have another PR since I'm not sure why Ankit removed those tests.

**Comment by kairu-ms** [comment]
> > 
> 
> Hello Kairu, thanks for the info!
> 
> If I understand it correctly, this PR needs tests to be added back. Are those generated when generating the cli command on the aaz ui?
> 
> Moreover, Ankit's aaz repo seems to have some conflicts that needs to be resolved and need another reviewer to review it. I don't seem to have the write access to it so I'm not sure what the conflict is.
> 
> If it's more convenient, I can generate the extensionimage command again and have another PR since I'm not sure why Ankit removed those tests.



> > 
> 
> Hello Kairu, thanks for the info!
> 
> If I understand it correctly, this PR needs tests to be added back. Are those generated when generating the cli command on the aaz ui?
> 
> Moreover, Ankit's aaz repo seems to have some conflicts that needs to be resolved and need another reviewer to review it. I don't seem to have the write access to it so I'm not sure what the conflict is.
> 
> If it's more convenient, I can generate the extensionimage command again and have another PR since I'm not sure why Ankit removed those tests.

Yes, you can regenerate the commands again. There are some tips can help, maybe we could have a quick call.

**Pull request closed without merging**
